build/Makefile

```diff
@@ -196,6 +204,12 @@ ifeq ($(ULINUX),1)
 endif
 
 #--------------------------------------------------------------------------------
+# Build AUOMS project
+
+$(AUOMS_TARGET_DIR)/auoms-bundle-test.sh : 
```

Comment:
I see,  having a comment will clear it out since it has the word "bundle" in the name.